src/client/v2/algod/models/types.ts

```diff
@@ -4571,7 +4678,7 @@ export class SimulateUnnamedResourcesAccessed extends BaseModel {
    * The unnamed applications that were referenced. The order of this array is
    * arbitrary.
    */
-  public apps?: (number | bigint)[];
```

Comment:
Oh, this is allowed?  I like that better than the Array stuff when an element is a union.

Reply:
is this syntax for an empty array if not specified?

Reply:
It is allowed and equivalent to `Array<number | bigint>`. I thought our formatter preferred the `Array<*>` syntax, but if this was here I'm doubting that now

Reply:
Well, I'll resolve. After all, it's deleted code.